docs/api.md

```diff
@@ -1656,6 +1656,17 @@ This function also returns a `Promise`, with `void` as its argument upon complet
 
 The new block will be placed at `referenceBlock.position.plus(faceVector)`.
 
+#### bot.placeEntity(referenceBlock, faceVector, cb)
```

Comment:
let's remove the cb (not adding it for new methods)

Reply:
Done